src/app/onboarding/components/create-new-archive/create-new-archive.component.spec.ts

```diff
@@ -4,25 +4,86 @@ import { CreateNewArchiveComponent } from './create-new-archive.component';
 import { OnboardingModule } from '@onboarding/onboarding.module';
 
 import { ArchiveVO } from '@models/archive-vo';
+import { ApiService } from '@shared/services/api/api.service';
+
+let calledCreate: boolean = false;
+let createdArchive: ArchiveVO | null;
+const mockApiService = {
+  archive: {
+    create: async (a: ArchiveVO) => {
+      calledCreate = true;
+      createdArchive = a;
+      return {
+        getArchiveVO: () => {
+          return a;
+        }
```

Comment:
This can be inlined:

```suggestion
        getArchiveVO: () => a,
```